src/pages/Categories.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Categories = () => {
+  return (
+    <div className='categories'>
+      <h2>Under construction</h2>
```

Comment:
- [ ] great job so far.. Everything is perfect. According to the project requirements, Kindly ensure the categories page should display only a button with text "Check status".